fearless/Common/View/IconWithSubtitleView.swift

```diff
@@ -0,0 +1,56 @@
+import UIKit
+
+class IconWithSubtitleView: UIView {
```

Comment:
Is it a common view or closely related to Beacon feature?

Reply:
I think it could be universal